consensus/rewards.md

```diff
@@ -0,0 +1,35 @@
+# Miners' rewards
+
+Aeternity is Proof of Work blockchain. This is a description of rewards paid to miners.
+
+## Introduction
+
+Miners mine key blocks and receive rewards composed out of two ingredients:
```

Comment:
This seems already described in [bitcoin-ng.md](https://github.com/aeternity/protocol/blame/c638d7728380bfdf85968dba1d22672f49664727/consensus/bitcoin-ng.md#L39_L46).